### DISCUSSION TITLE
Stars Count Bug on My Account?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#19808] [Projects and Issues]
I gave 4 stars to the repository but in the menu, it says only 1.
I've tried changing browsers and different devices but the problem remains the same.

![firefox_xGuxJfnMgH](https://user-images.githubusercontent.com/107854966/176859490-6d96f4c1-e3a3-4567-854c-55bdbc24d98f.jpg)

**Comment by icdance** [comment]
The count of the stars on GitHub doesn't seem up to date. May check back later, it usually fixes by itself.

[source](https://github.community/t/github-star-count-is-incorrect/193849)

**Comment by JhonnyLusonode** [comment]
Hello @wahyuwiyoko there is already a discussion about this issues in #19612

Check it out maybe there are all the answers you need.
In my opinion I believe it´s a bug... kindly report it to [Github Bug Report](https://support.github.com/contact/bug-report) over here if you believe it´s a bug too.

If you find this answer useful please consider marking it as "Answered".
(btw, this would help me to get the badge. Thank you! 😁)
